### ISSUE TITLE
Proto library rule errors out within a build definition

### ISSUE THREAD

**Issue by PeterBocan** [thought-machine/please#1213]
Hello guys, 
I am currently playing with Please v15.3.0 on Ubuntu 20.04 LTS. I have tried the following code 
```python
proto_library(
    name="proto",
    srcs=glob(["*.proto"]),
)
```
fails with the following error

```
plz build 
Build stopped after 540ms. 2 targets failed:
    //app/proto:all
rules/proto_rules.build_defs:232:25: error: Function filter_srcs can only be called with keyword arguments
                srcs = srcs,
                hdrs = [filter_srcs(name + '_hdrs', srcs[0], "h")],
                        ^
                deps = deps,

Traceback:
                       rules/proto_rules.build_defs:232:25:                   hdrs = [filter_srcs(name + '_hdrs', srcs[0], "h")],

                       rules/proto_rules.build_defs:229:56:               func = lambda name, srcs, deps, test_only: cc_library(
                                                   :  0: 0:   <source unavailable>
                       rules/proto_rules.build_defs: 62:30:           provides[language] = plugin['func'](
                       rules/proto_rules.build_defs: 60: 5:       for language, plugin in lang_plugins:
/home/peter/Projects/app/proto/BUILD:  2: 1:   proto_library(
```

I am not sure how to mitigate this issue, or if it's my mistake. I am trying to generate the golang library anyway, but this just popped up for me.

Thanks!
Peter.

**Comment by Tatskaari** [comment]
Ah sorry this is a bug with c/c++ protobuffs. If you're not using c or c++ you can set something like:
```
[proto]
language = go
language = py
language = java
language = js
```
depending on what languages you are using. I will get a fix out ASAP though.

**Comment by Tatskaari** [comment]
Fixed in #1216
